View/table_panel.py

```diff
@@ -0,0 +1,29 @@
+from PySide6.QtWidgets import QWidget, QPushButton, QGridLayout, QSizePolicy
+
+from View.encoding_table import EncodingTable
+
+
+class TablePanel(QWidget):
+    """Container of all Encoding Table related widgets."""
+
+    def __init__(self):
+        """
+        Constructor - Creates the related Encoding Table widgets and adds them
+        to the panel using a QGridLayout.
+        """
+        super().__init__()
+
+        self.table = EncodingTable()
+        self.add_col_button = QPushButton("Add column")
+        self.add_row_button = QPushButton("Add row")
```

Comment:
Should column and row be capitalized as it is a name of the button. Up to you to decide